targets/XPlatCppSdk/source/build/Xbox/TestXboxApp/TestXboxApp.vcxproj.ejs

```diff
@@ -153,34 +172,58 @@
     </ClCompile>
     <FXCompile>
       <ShaderModel>5.0</ShaderModel>
     </FXCompile>
+    <CustomBuildStep>
+      <Command>copy $(PF_TEST_TITLE_DATA_JSON) $(Platform)\Layout\Image\Loose /Y</Command>
+    </CustomBuildStep>
+    <CustomBuildStep>
+      <Message>Copying testTitleData.json for packaging</Message>
+      <TreatOutputAsContent>true</TreatOutputAsContent>
+      <Outputs>$(Platform)\Layout\Image\Loose\testTitleData.json</Outputs>
+    </CustomBuildStep>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClInclude Include="StepTimer.h" />
-    <ClInclude Include="Game.h" />
-    <ClInclude Include="pch.h" />
-    <ClInclude Include="TestCode.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\PlayFabApiTest.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\PlayFabEventTest.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\PlayFabTestMultiUserInstance.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\PlayFabTestMultiUserStatic.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestApp.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestAppPch.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestCase.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestContext.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestDataTypes.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestReport.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestRunner.h" />
+    <ClInclude Include="$(TestSourceDir)\TestApp\TestUtils.h" />
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="Main.cpp" />
-    <ClCompile Include="Game.cpp" />
-    <ClCompile Include="pch.cpp">
+    <ClCompile Include="$(TestSourceDir)\TestApp\PlayFabApiTest.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\PlayFabEventTest.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\PlayFabTestMultiUserInstance.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\PlayFabTestMultiUserStatic.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestApp.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestAppPch.cpp">
+      <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Debug|Durango'">Create</PrecompiledHeader>
       <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Release|Durango'">Create</PrecompiledHeader>
       <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Profile|Durango'">Create</PrecompiledHeader>
-      <PrecompiledHeader Condition="'$(Configuration)|$(Platform)'=='Debug|Durango'">Create</PrecompiledHeader>
     </ClCompile>
-    <ClCompile Include="TestCode.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestContext.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestReport.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestRunner.cpp" />
+    <ClCompile Include="$(TestSourceDir)\TestApp\TestUtils.cpp" />
+    <ClCompile Include="..\MainXBO.cpp" />
+    <ClCompile Include="..\TestAppXBO.cpp" />
```

Comment:
Minor ask that i overlooked in the initial review (in Beta repo): can we please move these files inside TestXboxApp project since they belong to it?